### DISCUSSION TITLE
新版网址增加全部字段搜索

### DISCUSSION THREAD

**Discussion by lllo-olll** [zotero-chinese/wiki#204] [好点子 / Ideas]
原页面可根据特征搜索，如“[A]. ”“~ ”“。见”等是我会经常使用的比较显著的条目特征，用来找符合样式的方法。
现新版网址仅可按标题搜索，建议新版网址在搜索栏目推出全部字段搜索，实现特征字段检索功能

**Comment by northword** [comment]
感谢建议，我们后续将添加更多条目特征标签，以及开放直接显示预览以方便通过 Ctrl + F 搜素，请追踪 https://github.com/zotero-chinese/website/issues/62 以获取更新进度。
